R/utils.R

```diff
@@ -1,12 +1,16 @@
 #' Utility function to calculate relative abundances
 #'
+#'
+#'
 #' @description
 #' The function calculates the column totals and divides each value within the
 #' column by the respective column total.
+#' 
+#' 
 #'
-#' @param se A SummarizedExperiment object with counts
+#' @inheritParams lefser
```

Comment:
Can you explain the purpose of this change? (I'm not sure what it does myself)

Reply:
I think the @param is used in documenting the parameter [se] which is the summarized experiment. That part was already on the code, I didn't make any changes to it.